packages/block-editor/src/components/media-upload/constants.js

```diff
@@ -1,15 +1,4 @@
-/**
- * WordPress dependencies
- */
-import { __ } from '@wordpress/i18n';
-
 export const MEDIA_TYPE_IMAGE = 'image';
 export const MEDIA_TYPE_VIDEO = 'video';
 export const MEDIA_TYPE_AUDIO = 'audio';
 export const MEDIA_TYPE_ANY = 'any';
-
-export const OPTION_TAKE_VIDEO = __( 'Take a Video' );
-export const OPTION_TAKE_PHOTO = __( 'Take a Photo' );
-export const OPTION_TAKE_PHOTO_OR_VIDEO = __( 'Take a Photo or Video' );
-export const OPTION_INSERT_FROM_URL = __( 'Insert from URL' );
-export const OPTION_WORDPRESS_MEDIA_LIBRARY = __( 'WordPress Media Library' );
```

Comment:
In reality, these same strings are [utilized elsewhere](https://github.com/WordPress/gutenberg/blob/c96b06aa947549c428e2e14a2d1d60ba1466516c/packages/block-editor/src/components/media-upload/index.native.js#L93-L141) outside of tests. An alternative to relocating these might be renaming this file to `constants.native.js` and replacing the duplicative string literals with references to these constants.

Reply:
@dcalhoun Yeah, this would be a good alternative. Probably if we rename the file to `constants.native.js` the i18n script will identify these strings as native-only ones. I'll apply your suggestion and check if the verify that the i18n script works as expected.